api-tests/features/disabled-cri-journeys.feature

```diff
@@ -0,0 +1,233 @@
+@Build
+Feature: Disabled CRI journeys
+
+  Rule: DCMAW is disabled
+
+    Scenario: A P1 journey takes the user on a no photo ID journey
+      Given I activate the 'dcmawOffTest,p1Journeys' feature sets
+      When I start a new 'low-confidence' journey
+      Then I get a 'page-ipv-identity-document-start' page response
+      When I submit an 'appTriage' event
+      Then I get a 'prove-identity-no-photo-id' page response with context 'nino'
+
+    Scenario: A P2 journey takes the user to the document select page
+      Given I activate the 'dcmawOffTest' feature set
+      When I start a new 'medium-confidence' journey
+      Then I get a 'page-ipv-identity-document-start' page response
+      When I submit an 'appTriage' event
+      Then I get a 'page-multiple-doc-check' page response
```

Comment:
This scenario is repeated in the below scenarios. If we keep the below (they might be too edge-casey), it might be worth removing this one.

Reply:
They are pretty edge-case-y, but I guess it's covering all the cases.

For now the tests seem to still be fairly speedy, but could be a good candidate for trimming...